### ISSUE TITLE
Chiarimento

### ISSUE THREAD

**Issue by gaiadobc** [iannellog/isi-analisi_di_log#4]

**Comment by iannellog** [comment]
i tre eventi principali li avevo suggeriti inizialmente al fine di visualizzare i cluster (se si visualizzano i punti in uno spazio 3D si può vedere se le features riescono a separare dei cluster o no)
per ora il task è calcolare più feature che possono potenzialmente caratterizzare gli utenti (in realtà sono per ora solo ipotesi di buon senso e un po' a caso) quindi vanno calcolate tutte
il problema di visualizzare il risultato per ora lo lasciamo da parte

**Comment by gaiadobc** [comment]
Va bene, lo chiedevo perché altrimenti nel vettore delle features che caratterizza ciascun utente per soddisfare la richiesta 'quante volte si è verificato ciascun evento' essendo gli eventi 42 verrebbe un vettore un po' scomodo (molto lungo).

**Comment by iannellog** [comment]
no, no non è lungo: è normale che in una prima fase si introducano molte feature (anche centinaia se non di più)
hai ragione nel notare che può essere scomodo da manipolare a mano: per questo dovremo poi capire che tipo di algoritmi usare per rendeei in qualche modo "visibile" il risultato a noi umani, ma questo è un problema successivo
